packages/fxa-content-server/app/tests/spec/views/settings/delete_account.js

```diff
@@ -26,7 +26,7 @@ describe('views/settings/delete_account', function() {
   const UID = '123';
   const password = 'password';
   let account;
-  let activeSubscriptions;
+  let subscriptions;
```

Comment:
I changed this var name as `subscriptions` is more accurate since it's the mock return for  `account.settingsData()`, before we've filtered for active subscriptions.